postgresql/values.yaml

```diff
@@ -64,7 +64,7 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  storageClass: netapp-block-standard
+  storageClass: netapp-file-standard
```

Comment:
let's try changing this back to `netapp-block-standard` and see if it's still blowing up; we really should use block storage for db if we can get it to provision properly

Reply:
still exploding

Reply:
![postgrestimeout](https://user-images.githubusercontent.com/26367840/77805239-a6fb6d00-703e-11ea-9085-25fcb20680f2.png)